contribs/drt/src/main/java/org/matsim/contrib/drt/optimizer/depot/Depots.java

```diff
@@ -64,13 +64,15 @@ public static boolean isSwitchingFromStopToStay(DvrpVehicle vehicle) {
 	}
 
 	public static Link findStraightLineNearestDepot(DvrpVehicle vehicle, Set<Link> links) {
```

Comment:
FYI Another option would be to pass a list of links to `findStraightLineNearestDepot()` instead of a set and shift the responsibility for reproducibility to the client code. But I think using the link ID for draws is safer.